packages/cli/test/utils/simulation/index.ts

```diff
@@ -0,0 +1,23 @@
+import {EventEmitter} from "events";
+import {TimeoutError} from "@lodestar/utils";
+
+export * from "./LodestarBeaconNodeProcess.js";
+export * from "./SimulationEnvironment.js";
+export * from "./utils.js";
+
+export function waitForEvent<T>(
+  emitter: EventEmitter,
+  event: string,
+  timeout = 3000,
+  condition: (e: T) => boolean = () => true
+): Promise<T> {
+  return new Promise((resolve, reject) => {
+    const timer = setTimeout(() => reject(new TimeoutError(`event ${event} not received`)), timeout);
+    emitter.on(event, (e) => {
```

Comment:
does this event handler need to be cleared?

Reply:
It was an unused code. Cleaned it.